src/app/header/header.component.html

```diff
@@ -63,7 +63,10 @@
         </ul>
       </li>
       <li routerLinkActive="active">
-        <a [routerLink]="['/work-item-list']" id="header_menuWork">Work</a>
+        <a [routerLink]="['/work-item/list']" id="header_menuWork">WorkList</a>
+      </li>
+      <li routerLinkActive="active">
+        <a [routerLink]="['/work-item/board']" id="header_menuWork">WorkBoard</a>
```

Comment:
It's just to navigate to the board view easily. It'll not be a part of the main project, since this section is a part of the platform UI.